README.md

```diff
@@ -1,3 +1,5 @@
 # contribute
 
 > hi this is testContribute user
+
+## contributing
```

Comment:
if you add some extra description I'll be happy

Reply:
I added more description 👍